met_connectors/src/lustre_netatmo/mod.rs

```diff
@@ -57,11 +58,19 @@ fn read_netatmo(time: Timestamp) -> Result<SpatialCache, data_switch::Error> {
             lats.push(record.lat);
             lons.push(record.lon);
             elevs.push(record.elev);
-            values.push(record.value);
+            values.push(Some(record.value));
         }
     }
 
-    Ok(SpatialCache::new(lats, lons, elevs, values))
+    Ok(DataCache::new(
+        lats,
+        lons,
+        elevs,
+        Timestamp(0),
+        RelativeDuration::minutes(5),
```

Comment:
You can use the real timestamp here from the function arguments. As for the time_resolution, these are all currently 1 hour. Eventually we plan to change that, but I think we will need to update this connector anyway when we do